src/main/java/use_cases/LoginManager.java

```diff
@@ -164,10 +164,10 @@ private boolean isValidUsername(String username) {
 
     }
 
-    private boolean isUniqueUser (String username){
+    private boolean isUniqueUser(String username) {
         User[] allUsers = databaseManager.getAllUsers();
-        for(User user: allUsers){
-            if(username.equals(user.getUsername())){
+        for (User user : allUsers) {
```

Comment:
Sorry to really nitpick, but I don't think you need the space after "user". This is just in terms of the style we currently have.